README.md

```diff
@@ -24,16 +25,27 @@ This module installs and configures [Kubernetes](https://kubernetes.io/). Kubern
 
 It groups containers that make up an application into logical units for easy management and discovery.
 
+## Install
+
+To use this module, add this declaration to your Puppetfile:
+```
+mod 'puppetlabs-kubernetes', '1.0.3'
+```
+To manually install this module with puppet module tool:
+```
+puppet module install puppetlabs-kubernetes --version 1.0.3
+```
+
 ## Setup
 
 ### Setup Requirements
 
-This module includes a configuration tool called `kubetool` to auto generate
-all the security parameters, the bootstrap token, and other configurations for
-your Kubernetes cluster into a Hiera file. The tool is available as a Docker
+This module includes a configuration tool called
+[kubetool](https://github.com/abema/kubetool) to auto generate all the security
```

Comment:
Not that kubetool :)
Our kubetool is specific to the module and in the tooling/kube_tool directory.